src/agent/application.cpp

```diff
@@ -370,6 +373,17 @@ void Application::InitNcpMode(void)
         });
     SetBorderAgentOnInitState();
 #endif
+#if OTBR_ENABLE_BACKBONE_ROUTER
+    mHost.SetBackboneRouterStateChangedCallback(
+        [this](otBackboneRouterState aState) { mMulticastRoutingManager->HandleStateChange(aState); });
+    mHost.SetBackboneRouterMulticastListenerCallback(
+        [this](otBackboneRouterMulticastListenerEvent aEvent, const Ip6Address &aAddress) {
+            mMulticastRoutingManager->HandleBackboneMulticastListenerEvent(aEvent, aAddress);
+        });
```

Comment:
I'm wondering if we should add the same registrations for RCP host?

Reply:
In this PR the multicast forwarding in RCP mode is unchanged. We will have a few dedicated PRs (including changes in openthread posix platform) to replace the implementation.